ikarus/finiteelements/fehelper.hh

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2021-2024 The Ikarus Developers mueller@ibb.uni-stuttgart.de
+// SPDX-License-Identifier: LGPL-3.0-or-later
+
+#pragma once
+
+#include <dune/localfefunctions/impl/standardLocalFunction.hh>
+#include <dune/localfefunctions/manifolds/realTuple.hh>
+
+#include <ikarus/finiteelements/ferequirements.hh>
+
+namespace Ikarus::FEHelper {
+  template <typename Traits, typename FERequirementType, typename DisplacementBasedElement, typename ScalarType>
+  auto getDisplacementFunctionImpl(const DisplacementBasedElement ele, const FERequirementType& par,
```

Comment:
This function should not be named `...Impl`. Furthermore, the templates should be deduces, thus using Traits as a template is not so nice. I think this function should receive a localview instead of "ele".